packages/api-request-builder/test/create-service.spec.js

```diff
@@ -66,7 +73,7 @@ describe('createService', () => {
     )
   })
 
-  it('should throw if definition prop (features) is a non empty array', () => {
+  it('should throw if definition prop (features) is an empty array', () => {
```

Comment:
🤔 why? `features` needs to be an array, so if it's **a non** empty array it throws